modules/control-disablements/main.tf

```diff
@@ -10,12 +10,18 @@ locals {
   is_central_cloudtrail_account = data.aws_caller_identity.this.account_id == var.central_logging_account && local.is_global_resource_region
   control_prefix                = "arn:${data.aws_partition.this.id}:securityhub:${data.aws_region.this.name}:${data.aws_caller_identity.this.account_id}:control"
 
+  # These controls are not supported in ap-northeast-3 (AN3), so we have to do some trickery to exclude them.
+  an3_excluded_controls = [
+    "${local.control_prefix}/aws-foundational-security-best-practices/v/1.0.0/IAM.4",
+    "${local.control_prefix}/cis-aws-foundations-benchmark/v/1.2.0/1.12",
+    "${local.control_prefix}/cis-aws-foundations-benchmark/v/1.2.0/1.20"
+  ]
+
```

Comment:
@mcalhoun 
maybe we can create a map of regions to lists of not supported controls?

```
not_supported_region_controls = {
   "ap-northeast-3" = [
     "${local.control_prefix}/aws-foundational-security-best-practices/v/1.0.0/IAM.4",
      "${local.control_prefix}/cis-aws-foundations-benchmark/v/1.2.0/1.12",
      "${local.control_prefix}/cis-aws-foundations-benchmark/v/1.2.0/1.20"
   ]
}
```

otherwise LGTM

Reply:
Honestly, it's just a broken region...everything else is supported everywhere.